requirements.txt

```diff
@@ -6,3 +6,4 @@ pytest
 pandas
 numba
 plum-dispatch
+torch == 2.3.0
```

Comment:
Hi Fionn, I encountered a problem adding this requirement: I added it to a file called `torch.txt` under the `dev` folder, and I also modified the `setup.py` file accordingly. But it cannot pass the automatic checks because it didn't detect the torch requirement. What should I do on this?

Reply:
solved by adding `-r torch.txt` to dev.txt

Reply:
That's right. Also, I wouldn't necessarily pin to a specific version, it can be helpful to give the resolver a bit of leeway (>= 2.3.0 is ok for example). In general you should guard imports for optional dependencies but adding to dev makes sense.

Reply:
OK thanks Fionn, I modified it to >= 2.3.0